### PR TITLE
Adds Kakerlakenbier (roachbeer)

### PR DESCRIPTION
**Pull request by Noobetski**
## About The Pull Request
<details>
<summary>
	Adds Kakerlakenbier (roachbeer). Without special glass sprite for now. Also fixes minor typo in bullet scanner
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:
add: Adds Kakerlakenbier AKA roach beer

/:cl:

### CODE REVIEW

**Comment by Noobetski** [comment]
Why it doesn't identifies strength when friendly fire also uses it?

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 1]
```diff
@@ -3156,3 +3156,15 @@
 	glass_icon_state = "friendlyfire"
 	glass_name = "Friendly Fire"
 	glass_desc = "Watch your back when you take a swig."
+
+/datum/reagent/alcohol/roachbeer
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagents/food-Drinks.dm:3160]:
```suggestion
/datum/reagent/ethanol/roachbeer
```

**Comment by Trilbyspaceclone** [comment]
Were is its icon?

**Comment by Noobetski** [comment]
Ah, i thought by making glass_unique_appearance false it would disable need for icon.
I'm not spriter sadly, so can't make new icon for it

**Comment by Noobetski** [comment]
On Eris it's just glass with chem color inside (probably because it counts as stimulant there due to vagabond sanity boost)

**Comment by cdb-is-not-good** [comment]
Interbreasting- probably should try to source a proper glass sprite if it's intended to be a proper drink rather than just a chem

File: code/modules/detectivework/tools/bullet_scanner.dm [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 /obj/item/device/bullet_scanner
 	name = "\improper Bullet Scanner"
-	desc = "A small device used to scan bullet holes or laser marks to get more data on the gun that fire it."
+	desc = "A small device used to scan bullet holes or laser marks to get more data on the gun that fired it."
+
```
Comment (nikothedude) [inline code/modules/detectivework/tools/bullet_scanner.dm:6]: what's up with this newline

Comment (Noobetski) [reply code/modules/detectivework/tools/bullet_scanner.dm:6]: Someone made typo. i changed it from gun that fire it to gun that fired it

Comment (Noobetski) [reply code/modules/detectivework/tools/bullet_scanner.dm:6]: actually i thought it already was PRed with other branch, but oh well

**Pull request closed without merging**